configuration/wireless/access-point-routed.md

```diff
@@ -0,0 +1,190 @@
+# Setting up a Raspberry Pi as a Routed Wireless Access Point
+
+The Raspberry Pi can be used as a wireless access point routing to an existing ethernet network. This will create a new wireless network entirely managed by the Raspberry Pi.
+
+If you wish to extend an existing ethernet network to wireless clients, consider instead setting up a [bridged access point.](./access-point-bridged.md)
+
+```
+                                         +- RPi -------+
+                                     +---+ 10.10.0.2   |          +- Laptop ----+
+                                     |   |     WiFi AP +-)))  (((-+ WiFi STA    |
+                                     |   | 192.168.4.1 |          | 192.168.4.2 |
+                                     |   +-------------+          +-------------+
+                 +- Router ----+     |
+                 | Firewall    |     |   +- PC#2 ------+
+(Internet)---WAN-+ DHCP server +-LAN-+---+ 10.10.0.3   |
```

Comment:
For consistency with the diagram on the other page, maybe it's worth switching the Ethernet network IPs to 192.168.1.x and then have 10.10.0.x on the WiFi network?

Reply:
Sorry, I didn't see that. I used 10.10.0.0 instead of 192.168.1.0 to make the contrast more apparent between the 2 nets. (192.168.4.1 and 192.168.1.1 differ less visually)

Could have been a bad idea...